apps/web/src/components/settings/sections/TerminalSection.tsx

```
@@ -33,6 +49,37 @@ export function TerminalSection() {
+            const isSelected = editorProtocol === editor.id;
+            return (
+              <button
+                key={editor.id}
+                onClick={() => setEditorProtocol(editor.id)}
+                className={cn(
+                  'px-3 py-1.5 rounded-md text-sm border transition-colors',
+                  isSelected
+                    ? 'bg-primary text-primary-foreground border-primary'
+                    : 'bg-card border-border text-foreground hover:bg-muted'
+                )}
+              >
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)